libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartStoreExternalStorageTest.java

```diff
@@ -91,28 +89,6 @@ public void testRegisterSoupWithExternalStorageAndJSON1() {
         Assert.assertFalse("Register soup call should have failed", store.hasSoup(OTHER_TEST_SOUP));
 	}
 
-	/**
-	 * Ensure data is still accessible after changing key
-	 */
-    @Test
-	public void testChangeKey() throws JSONException {
```

Comment:
Why kill that test?

Reply:
I didn't want to add public APIs for `legacyEncrypt` - it adds too many public APIs that apps might call and there's very little value in adding it because a simple manual test exposes whether it works or not - there's no halfway. Since there's no `legacyEncrypt`, I'd have to hardcode that in this test - not sure if there's enough value in it, but I can add it back if we want to.

Reply:
@wmathurin Brought back the test below. No code changes - just moved down in the file.